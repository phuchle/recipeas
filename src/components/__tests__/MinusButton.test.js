import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MinusButton from '../MinusButton';

describe('MinusButton', () => {
  test('renders without crashing', () => {
    const component = shallow(<MinusButton />);
    const tree = toJson(component);

    expect(tree).toMatchSnapshot();
  });
});
