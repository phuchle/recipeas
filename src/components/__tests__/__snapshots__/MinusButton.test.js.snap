// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MinusButton renders without crashing 1`] = `
<Button
  active={false}
  block={false}
  bsClass="btn"
  bsStyle="danger"
  className="undefined glyphicon glyphicon-minus"
  disabled={false}
  style={
    Object {
      "marginLeft": "5px",
    }
  }
/>
`;
